chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -497,7 +497,11 @@ Zotero.Utilities.Internal = {
 				item.linkMode = zoteroItem.attachmentLinkMode;
 				item.mimeType = item.contentType;
 			}
-			
+			// preserve notes
+			if(item.note) { 
+				item.uniqueFields.note = item.note;
+			}
+
```

Comment:
OK, could you add a space after `if` then? I'd also prefer to move this next to other unique fields (L456)